system-tests/FortranDependencies/first.f90

```diff
@@ -2,7 +2,7 @@
 ! For further details please refer to the file COPYRIGHT
 ! which you should have received as part of this distribution
 !
-program first
+program first_mod
```

Comment:
Why did you rename this to `first_mod`?  I guess to highlight that the unit name needn't match the source filename?  Which is fine, I just wonder about using `_mod` - which is conventionally used for modules

Reply:
Crikey! That was a mistake. `first` is a program so should never have the `_mod` suffix. Having program units which don't match their filenames is a good call though. I've renamed "second" to "two".

Reply:
Easily done!